### PR TITLE
gzip for log, minor consistency formatting

### PR DESCRIPTION
**Pull request by aretecode**
- [gzip in log file](https://github.com/fuse-box/fuse-box/compare/master...aretecode:gzip?expand=1#diff-ecd92f5709aad339d83c281d5a01477dR147)
- the rest is minor multi-multi-multi empty lines that follow the common consistent formatting

### CODE REVIEW

**Comment by nchanged** [comment]
The tests have failed, please don't commit `minor formatting` i am not going to merge 42 files. That's insane.

**Comment by aretecode** [comment]
@nchanged 
- tests only failed for minor lint of *unused* code for *todo*
- look at the lines changed, only ~100, they are ONLY empty lines, I put it in description, just while I was searching for the best place to extract the bundle information from to keep consistency

**Comment by nchanged** [comment]
@aretecode regardless, please discard it.

File: src/Log.ts [hunk 1]
```diff
@@ -133,8 +140,26 @@ export class Log {
         this.echoBundleStats(header || "Bundle", this.totalSize, took);
     }
 
+    /**
+     * @TODO
+     *  - [ ] ensure header will not conflict if it is used in echoBundleStats
+     */
+    public echoGzipSize(size: string | number) {
```
Comment (nchanged) [inline src/Log.ts]: so where is it being used?

**Pull request merged**